src/index.tsx

```diff
@@ -1482,6 +1567,231 @@ class App extends React.Component<{}, AppState> {
               Load file...
             </button>
           </div>
+          {someElementIsSelected() && (
```

Comment:
Can you put it back next to the top, next to the other selected shapes options?

Reply:
Only the colors options? Or all the properties?

Reply:
All of them.